app-frontend/src/app/components/pages/project/index.js

```diff
@@ -49,5 +50,6 @@ export default [
     projectPage,
     ...layerPages,
     ...settingsPages,
-    ...analysesPages
+    ...analysesPages,
+    navbar
```

Comment:
I've been using `.../pages` as a place to store routed components, but I think the belongs under `app/components/common` instead
see `app/components/projects/index.js` and `app/components/projects/layerItem/...` and how I imported them into index.components.js to make imports a bit more sane